packages/runtime/container-runtime/.eslintrc.cjs

```diff
@@ -46,6 +50,30 @@ module.exports = {
 		"jsdoc/multiline-blocks": ["error", { noSingleLineBlocks: true }],
 		"jsdoc/require-description": ["error", { checkConstructors: false }],
 
+		"unicorn/catch-error-name": "error",
+		"unicorn/consistent-destructuring": "error",
+		"unicorn/new-for-builtins": "error",
+		"unicorn/no-array-callback-reference": "error",
+		"unicorn/no-array-for-each": "error",
+		"unicorn/no-null": "error",
+		"unicorn/no-zero-fractions": "error",
+		"unicorn/prefer-node-protocol": "error",
+		"unicorn/prefer-number-properties": "error",
+		"unicorn/prefer-optional-catch-binding": "error",
+		"unicorn/prefer-spread": "error",
+		"unicorn/prefer-string-slice": "error",
+		"unicorn/switch-case-braces": "error",
+		"unicorn/throw-new-error": "error",
+
+		// TODO:
```

Comment:
Future PR (trying to keep these PRs somewhat scoped so they're easier to review)